### PR TITLE
Update translation into Spanish

### PR DESCRIPTION
**Pull request by nvdaes**
## Summary of changes
Updated translation into Spanish.

### Pull Request Checklist
- [ ] Changes have tests
- [x] News fragment added in news.d. See [documentation](https://github.com/openstenoproject/plover/blob/master/doc/developer_guide.md#making-a-pull-request) for details

### CODE REVIEW

**Comment by nvdaes** [comment]
@benoit-pierre , I've updatet the spanish translation as you requested in PR #1258.
Thanks.

File: plover/messages/es/LC_MESSAGES/plover.po [hunk 1]
```diff
@@ -2,191 +2,262 @@ msgid ""
...
+#. Widget: “AddTranslationWidget”.
+#: plover/gui_qt/add_translation_widget.py:274
 msgid "{strokes} is not mapped in any dictionary"
 msgstr "{strokes} no está asignada en ningún diccionario"
 
-#: plover/gui_qt/add_translation_widget.py:287
+#. Widget: “AddTranslationWidget”.
+#: plover/gui_qt/add_translation_widget.py:290
 msgid "{translation} is mapped to: {strokes}"
 msgstr "\"{translation}\" está asignada a {strokes}"
```
Comment (benoit-pierre) [inline plover/messages/es/LC_MESSAGES/plover.po:99]: Are the extra `"` necessary? The one right below does not have them.

**Review by benoit-pierre — COMMENTED** [review]
Looking good, thanks for updating!

File: plover/messages/es/LC_MESSAGES/plover.po (hunk 1 above)
Comment (nvdaes) [reply plover/messages/es/LC_MESSAGES/plover.po:99]: > 
> 
> Are the extra `"` necessary? The one right below does not have them.


No, it was a mistake from my part. You can request changes (adding a suggestion with control+g) so I can accept the suggestion from GitHub or I'll fix the file and update it.
Thanks

Comment (benoit-pierre) [reply plover/messages/es/LC_MESSAGES/plover.po:99]:
```suggestion
msgstr "{translation} está asignada a {strokes}"
```

**Review by benoit-pierre — CHANGES_REQUESTED** [review]

**Review by benoit-pierre — APPROVED** [review]

**Comment by nvdaes** [comment]
Thanks @benoit-pierre for aplying the suggestion.

**Pull request merged**